seeds/01_projects.js

```diff
@@ -0,0 +1,13 @@
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('projects').truncate()
+    .then(function () {
+      // Inserts seed entries
+      return knex('projects').insert([
+        {id: 1, name: 'Gutters', description: 'Clean them gutters out homie', completed: false},
+        {id: 2, name: 'Clean Car', description: 'Clean inside and out of your Toyota', completed: false},
```

Comment:
Why explicitly use `id` here? Your `increments` from your migration along with `truncate` should take care of it.